plugins/destination/snowflake/client/client.go

```diff
@@ -38,11 +37,18 @@ func New(ctx context.Context, logger zerolog.Logger, destSpec specs.Destination)
 	if err := spec.Validate(); err != nil {
 		return nil, err
 	}
-	_, err := gosnowflake.ParseDSN(spec.ConnectionString)
+	cfg, err := gosnowflake.ParseDSN(spec.ConnectionString)
 	if err != nil {
 		return nil, err
 	}
-	db, err := sql.Open("snowflake", spec.ConnectionString)
+	binaryFormat := "BASE64"
```

Comment:
This seems like an additional breaking change? Can we add it to the change log as well? See example https://github.com/cloudquery/cloudquery/pull/10169 for multiple breaking changes in a single PR

Reply:
> This seems like an additional breaking change? Can we add it to the change log as well? See example #10169 for multiple breaking changes in a single PR

@erezrokah I think this is about how Arrow encodes binary columns.

Reply:
It's not really a breaking change because it's an encoding for the session only i.e for the arrow/json format when we insert.

Reply:
Cool, I thought this impacts the format of the data saved in the DB https://docs.snowflake.com/en/user-guide/binary-input-output#base64